qiita_pet/handlers/api_proxy/artifact.py

```diff
@@ -53,3 +57,136 @@ def artifact_graph_get_req(artifact_id, direction, user_id):
             'node_labels': node_labels,
             'status': 'success',
             'message': ''}
+
+
+def artifact_get_req(artifact_id, user_id):
+    """Get information about the artifact
+
+    Parameters
+    ----------
+    artifact_id : int
+        Artifact being acted on
+    user_id : str
+        The user requesting the action
+
+    Returns
+    -------
+    dict
+        information about the artifact
+    """
+    pd = Artifact(int(artifact_id))
+    access_error = check_access(pd.study.id, user_id)
+    if access_error:
+        return access_error
+
+    try:
+        can_be_submitted_to_ebi = pd.can_be_submitted_to_ebi
+        ebi_run_accessions = pd.ebi_run_accessions
+    except QiitaDBOperationNotPermittedError:
+        can_be_submitted_to_ebi = False
+        ebi_run_accessions = None
+
+    try:
```

Comment:
Also I don't see tests in which can_be_submitted_to_XXX gets true/false (only false are tested).